.azure/pipelines/jobs/codesign-xplat.yml

```diff
@@ -21,11 +21,21 @@ jobs:
         artifactName: ${{ parameters.inputName }}_Packages
         downloadPath: $(Build.StagingDirectory)/deps/
         itemPattern: '**/*.nupkg'
+    - script: powershell -ExecutionPolicy ByPass -NoProfile build.ps1 /t:Restore
+      displayName: Restore tools
```

Comment:
1. checked locally and .\restore.cmd (the equivalent of this command) does not restore anything for the XplatPackageSigner.proj project
    - suggest adding `-restore` to the `msbuildArguments` below
2. The XplatPackageSigner.proj project does not have a 'restore' target
    - might need to rename the file to XplatPackageSigner.**cs**proj and maybe make a few other changes e.g. add `<IsPackable>false</IsPackable>`, `<IsProjectReferenceProvider>false</IsProjectReferenceProvider>` and `<IsShipping>false</IsShipping>`
    - alternatively, might be able to `<import/>` the correct NuGet files to add a 'restore' target